docfiles/style.css

```diff
@@ -58,6 +58,19 @@ li code:before, li code:after {
     content: "\00a0";
 }
 
+table {
```

Comment:
should this go in docs.less?

Reply:
I moved the styles into theme/docs.less
Out of curiosity, how do you recompile the styles from less in pxt? I ended up having to setup another local server just to test if it worked.